lib/timeout.rb

```diff
@@ -76,9 +76,15 @@ def exception(*)
   # Note that this is both a method of module Timeout, so you can <tt>include
   # Timeout</tt> into your classes so they have a #timeout method, as well as
   # a module method, so you can call it directly as Timeout.timeout().
```

Comment:
Should the documentation be updated to explain this new behavior?

Reply:
Yes good idea.